device/dell/x86_64-dellemc_z9432f_c3758-r0/plugins/fanutil.py

```diff
@@ -20,26 +20,30 @@ class FanUtil(FanBase):
     NUM_FANS_PERTRAY = 2
     FANTRAY_NUM_START_IDX = 1
     FRU_FAN_START_IDX = 1
-    IPMI_FAN_PRESENCE = "ipmitool sensor get FAN{0}_prsnt"
-    IPMI_FAN_FRONT_SPEED = "ipmitool sdr get Fan{0}_Front_rpm"
-    IPMI_FAN_REAR_SPEED = "ipmitool sdr get Fan{0}_Rear_rpm"
-    IPMI_FRU_DATA = "ipmitool fru print {0}"
+    IPMI_FAN_PRESENCE = ["ipmitool", "sensor", "get", "FAN"]
```

Comment:
>FAN [](http://example.com/codeflow?start=55&length=3)

Suggest
1. Initial it with `FAN{0}_prsnt`
2. self.IPMI_FAN_PRESENCE[3] = self.IPMI_FAN_PRESENCE[3].format(fan_id)

So the logic is more similar to old code.

Reply:
Updated, thanks.

Reply:
Sorry for misleading. This will lead to a bug if user call `get_fan_status()` multiple times.